blocks/youtube-player/youtube-player.js

```diff
@@ -1,37 +1,45 @@
 function loadVideo(videoURL, block) {
-  const VIDEO_HOST = 'https://www.youtube.com';
   const overlay = document.createElement('div');
-  overlay.classList.add('video-player-overlay');
+  overlay.classList.add('youtube-player-overlay');
   block.parentElement.appendChild(overlay);
 
   const toolbar = document.createElement('div');
-  toolbar.classList.add('video-player-toolbar');
+  toolbar.classList.add('youtube-player-toolbar');
   const toolbarClose = document.createElement('div');
-  toolbarClose.classList.add('video-player-close');
+  toolbarClose.classList.add('youtube-player-close');
   toolbar.appendChild(toolbarClose);
   block.parentElement.appendChild(toolbar);
 
   const videoIframe = document.createElement('iframe');
-  videoIframe.classList.add('video-player-iframe');
+  videoIframe.classList.add('youtube-player-iframe');
   videoIframe.setAttribute('allowfullscreen', '');
-  videoIframe.src = `${VIDEO_HOST}/embed${videoURL.pathname}`;
+  videoIframe.src = `https://www.youtube.com/embed${videoURL.pathname}`;
 
   block.appendChild(videoIframe);
-
-  overlay.addEventListener('click', () => {
+  function escEvent(e) {
+    if (e.key === 'Escape' || e.key === 'Esc') {
+      // eslint-disable-next-line no-use-before-define
```

Comment:
no unnecessary eslint disables.... please move the `removeVideo` function accordingly.

Reply:
that seems circular design

Reply:
no, because in one of the functions, the first function is only needed for reference to remote the event listener. But it does not actually call it

Reply:
then use a `fn` argument for the `removeVideo` ...